cmd/ctl/pkg/upgrade/migrateapiversion/migrator.go

```diff
@@ -175,7 +175,8 @@ func (m *Migrator) migrateResourcesForCRD(ctx context.Context, crd *apiext.Custo
 		return err
 	}
 	fmt.Fprintf(m.Out, " %d resources to migrate...\n", len(list.Items))
-	for _, obj := range list.Items {
+	for i := range list.Items {
+		obj := list.Items[i]
```

Comment:
False positive because obj was only passed to the synchronous `client.Update` method inside the loop.